### ISSUE TITLE
DXVK 0.95: Kingdom Come missing resolutions 1440/1600

### ISSUE THREAD

**Issue by jarrard** [doitsujin/dxvk#867]
Just wondering if anyone has been having issues getting 1440p and 1600p resolution options to show for this game? I remember having this issue before and was wondering if its related to me running at 4k and having a 1080p secondary screen to the side?

It will be a real pita if I must goto 1440p desktop just to change to that ingame. At present my options are 1200p directly to 2160p for high resolution options. 1080TI isn't enough for 4k under Linux for many games like this so I can't just run at 2160p, unless I like 30-40fps dips at every turn.

### Software information
Kingdom Come Deliverance, Medium to high.

### System information
- GPU: 1080TI
- Driver: 415.22-05
- Wine version: 3.21 Protonified 
- DXVK version: 0.95

### Apitrace file(s)
- Not needed

### Log files
- d3d11.log:
- dxgi.log:

**Comment by jarrard** [comment]
I did some more testing and it seems just using wine4.0 staging and not a pkg protonified version allows 1440p as a option. Also if you set lutris to restore resolution, and minimize while in game at 1440p, it will fix the desktop to 4k while the game runs at 1440p.. odd but it works.

Guess PKG wine builds have some issues perhaps.  I'll close this for now since I figured out a workaround and it doesn't appear to be directly DXVK related.
